CompanyApiTest/CompanyApiTest.cs

```diff
@@ -16,6 +18,145 @@ public CompanyApiTest()
             httpClient = webApplicationFactory.CreateClient();
         }
 
+        [Fact]
+        public async Task Should_return_success_code_when_delete_employee_successfully_given_companyId_and_employeeId()
+        {
+            //given
+            await ClearDataAsync();
+            CreateCompanyRequest companyGiven = new CreateCompanyRequest("BlueSky Digital Media");
+            HttpResponseMessage postResponseMessage = await httpClient.PostAsJsonAsync("/api/companies", companyGiven);
+            var createdCompany = await postResponseMessage.Content.ReadFromJsonAsync<Company>();
+            var createdEmployee = new EmployeeRequest("pengyu");
+            HttpResponseMessage postResponseMessage2 = await httpClient.PostAsJsonAsync($"/api/companies/{createdCompany?.Id}/employees", createdEmployee);
+            var createdEmployee2 = await postResponseMessage2.Content.ReadFromJsonAsync<Employee>();
+
+            //when
+            HttpResponseMessage httpResponseMessageFinal = await httpClient.DeleteAsync($"/api/companies/{createdCompany?.Id}/employees/{createdEmployee2?.Id}");
+
+            //then
+            Assert.Equal(HttpStatusCode.NotFound, httpResponseMessageFinal.StatusCode);
+        }
+
+        [Fact]
+        public async Task Should_return_success_code_when_add_employee_to_the_company_given_companyId_and_new_Employee()
+        {
+            //given
+            await ClearDataAsync();
+            CreateCompanyRequest companyGiven = new CreateCompanyRequest("BlueSky Digital Media");
+            HttpResponseMessage postResponseMessage = await httpClient.PostAsJsonAsync("/api/companies", companyGiven);
+            var createdCompany = await postResponseMessage.Content.ReadFromJsonAsync<Company>();
+            var createdEmployee = new EmployeeRequest("pengyu");

```

Comment:
Suggestion: use givenEmployee when creating an employee request; use createdEmployee read from postResponseMessage because that's successfully created.